src/Pages/Auth/Login/index.js

```diff
@@ -1,11 +1,33 @@
-import React from "react"
+import React, { useEffect } from "react"
+import { Link } from "react-router-dom"
 
 Login.propTypes = {}
 
-function Login() {
+function Login(props) {
+  const loggedStatous = props.userIsLogged
```

Comment:
no need to copy this into a local variable. You can just use `props.userIsLogged`